components/login/login.scss

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+@import 'utils/mixins';
+
+.login-body {
+    display: flex;
+    flex: 1;
+    align-items: center;
+
+    .login-body-content {
+        display: flex;
+        height: fit-content;
+        flex: 1;
+        align-items: center;
+        justify-content: center;
+
+        .login-body-no-login {
+            display: flex;
+            max-width: 540px;
+            flex-flow: column;
+            justify-content: center;
+
+            .login-body-no-login-title {
+                color: #1e325c;
```

Comment:
do we have a css variable for this? if not there are several occurrences of this hex in the PR too, maybe makes sense to create one